### PR TITLE
views: json based interface

### PR DESCRIPTION
**Pull request by jacquerie**
- Convert GROBID's TEI XML output to a simplified JSON
  representation, and use that to show the result to the user.

Signed-off-by: Jacopo Notarstefano jacopo.notarstefano@cern.ch

### CODE REVIEW

File: invenio_grobid/utils.py [hunk 1]
```diff
@@ -26,7 +26,89 @@
```
Comment (kaplun) [inline invenio_grobid/utils.py:26]: As in general, this should almost live into our overlay. What if we move this to a `contrib` sub dir and load it via the usual  `werkzeug` `import_string` pattern? (following a some config var in `invenio-grobid.config`?

Comment (jalavik) [inline invenio_grobid/utils.py:26]: In general. I agree. E.g. the upload hook is done like this already (in this file there is just a generic sample). We can add a similar hook for the conversion. However, I'd suggest provide a "generic" conversion here, out-of-the-box, that people can use.

Comment (jacquerie) [inline invenio_grobid/utils.py:26]: Eh. This code was written in a hurry to have a functioning demo, but is code I wouldn't recommend merging: this _has_ to have a better solution than the current madness.

**Comment by jacquerie** [comment]
Ping @jalavik.

Build is not passing, but I'm getting a different error locally. It shouldn't really matter, since it's something we currently don't use. WIll add a `@pytest.skip` and repush.

**Comment by jacquerie** [comment]
PEP8 fixed, build is now passing.

**Pull request merged**